views/site/index.php

```diff
@@ -9,4 +8,21 @@
         <h3>You are on PHIS, the Hybrid Information System about Phenotyping !</h3>
         <!--<h3>Yii::t('app/messages', 'Intro'); </h3>-->
     </div>
+    <?php    
+//    SILEX:info
+//    If you want to activate animated multiple background images 
+//    feature uncomment these lines below and add
+//    'use app\components\widgets\FullScreenImageSliderWidget;'
+//    at the beginning of the file. You can also see full-slider.css
+//    file to set different annimation style.
+//    \SILEX:info
+//        echo FullScreenImageSliderWidget::widget([
```

Comment:
Can we choose the slider duration when calling the widget ?

Reply:
Yes, can do it by setting it in widget options (for more information look at the FullScreenImageSliderWidget.php file).
In the code below the time between the transition of two images takes 10 seconds.
```php
FullScreenImageSliderWidget::widget([
            FullScreenImageSliderWidget::DURATION_PER_IMAGE => 10,
            FullScreenImageSliderWidget::IMAGES_URL_LINK => [
               "background/wallpaper_grapes_vine.jpg",
               "background/wallpaper_leaf.jpg",
               "background/wallpaper_tomato.jpg",
               "background/wallpaper_vine.jpg"
          ]
       ]);
```

Reply:
Could you add some information on the documentation ?